definitions/region/common.yaml

```diff
@@ -130,76 +131,80 @@
         Trinidad and Tobago, United States Virgin Islands, Uruguay, Venezuela
   - Other (R9):
       description: Rest of the World, used only if a match with the R9 regions can
-        otherwise not be achieved.
+        otherwise not be achieved
       navigate: R9OWO
 
 # The R10 definitions were introduced by the LIMITS project
 # funded by the EU's Seventh Framework Programme FP7/2007-2013
 # (grant agreement no. 282846)
```

Comment:
```suggestion
# (grant agreement no. 282846) https://tntcat.iiasa.ac.at/LIMITSDB/static/download/LIMITS_10+1_regions.pdf
```

Reply:
We don't know how much longer these databases will be accessible, so I'd rather not add a static link